plugins/kubernetes/api-report.md

```diff
@@ -259,6 +261,12 @@ export interface KubernetesApi {
   getWorkloadsByEntity(
     request: WorkloadsByEntityRequest,
   ): Promise<ObjectsByEntityResponse>;
+  // (undocumented)
+  proxy(options: {
```

Comment:
I think it would be nice for the proxy to be it's own API that we could add common methods to (like fetching logs etc)?

Reply:
Yeah, this is something we can add later or I can take a first pass at it in my logging PR. I think it makes sense to merge this PR first and me to rebase in that case